### PR TITLE
Serverless FAQ and limitations

### PR DESCRIPTION
**Pull request by SunRunAway**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/serverless-tier-faqs.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+### What is Serverless Tier?
+
+TiDB Cloud Serverless Tier delivers the free starter TiDB clusters with full HTAP capabilities for you and your organization. It is a fully managed, auto-scaling deployment of TiDB that lets you start using your database immediately, develop and run your application without caring about the underlying nodes, and automatically scale based on your application's workload changes.
+
+### How do I get started with Serverless Tier?
+
+Get started with the 5-minute [TiDB Cloud Quick Start](/tidb-cloud/tidb-cloud-quickstart.md).
+
+### What are the limitations of a Serverless Tier cluster?
```
Comment (ran-huang) [inline tidb-cloud/serverless-tier-faqs.md:22]: 这里建议 link 到 tidb-cloud/serverless-tier-limitations.md

File: tidb-cloud/serverless-tier-limitations.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+summary: Learn about the limitations of TiDB Cloud Serverless Tier.
+---
+
+# Serverless Tier Limitations
+
+<!-- markdownlint-disable MD026 -->
+
+Serverless Tier has some limitations below. We are also constantly filling in the feature gaps between Serverless Tier and Dedicated Tier. If you require these features or capabilities, please use [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) or [contact us](https://www.pingcap.com/contact-us/?from=en) for a feature request.
+
+## General Limitations
```
Comment (qiancai) [inline tidb-cloud/serverless-tier-limitations.md:12]:
```suggestion
## General limitations
```

File: tidb-cloud/serverless-tier-limitations.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- For each TiDB Cloud account, you can create one complimentary Serverless Tier cluster to use during the beta phase. You need to delete the existing Serverless Tier cluster then create a new one.
```
Comment (qiancai) [inline tidb-cloud/serverless-tier-limitations.md:14]:
```suggestion
- For each TiDB Cloud account, you can create one complimentary Serverless Tier cluster during the beta phase. To create a new Serverless Tier cluster, you need to delete the existing one first.
```

File: tidb-cloud/serverless-tier-limitations.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+- For each TiDB Cloud account, you can create one complimentary Serverless Tier cluster to use during the beta phase. You need to delete the existing Serverless Tier cluster then create a new one. 
+- Each Serverless Tier cluster has the following limitations:
+    - The storage size is limited to 5GiB (logical size) of OLTP storage and 5GiB of OLAP storage.
+    - The compute resource is limited to 1 vCPU and 1GiB RAM.
+    - The data transfer is limited to a total of 20 GiB in and out per week. If the 20 GiB limit is reached, the network traffic will be throttled to 10 KB/s.
+
+## Connection
+
+- Only [Standard Connection](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-standard-connection) can be used. You cannot use [Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections.md) or [VPC Peering](/tidb-cloud/set-up-vpc-peering-connections.md) to connect to Serverless Tier clusters. 
+- No "IP Access List" support.
```
Comment (qiancai) [inline tidb-cloud/serverless-tier-limitations.md:23]:
```suggestion
- IP Access List is not supported, which means you cannot specify which IP addresses are allowed to connect to your cluster.
```

File: tidb-cloud/serverless-tier-limitations.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+## Backup and Restore
+
+- [Backup and Restore](/tidb-cloud/backup-and-restore.md) are not supported currently.
```
Comment (qiancai) [inline tidb-cloud/serverless-tier-limitations.md:27]:
```suggestion
- [Backup and Restore](/tidb-cloud/backup-and-restore.md) are not supported for Serverless Tier currently.
```

File: tidb-cloud/serverless-tier-limitations.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+## Monitoring
+
+- [Built-in Monitoring](/tidb-cloud/built-in-monitoring.md) is currently not available for Serverless Tier.
+- [Third-party Monitoring integrations](/tidb-cloud/third-party-monitoring-integrations.md) is currently not available for Serverless Tier.
+
+## Others
+
+- [Open API](/tidb-cloud/api-overview.md) and [Terraform Integration](/tidb-cloud/terraform-tidbcloud-provider-overview.md) are currently not available.
```
Comment (qiancai) [inline tidb-cloud/serverless-tier-limitations.md:36]:
```suggestion
- [Open API](/tidb-cloud/api-overview.md) and [Terraform Integration](/tidb-cloud/terraform-tidbcloud-provider-overview.md) are currently not available for Serverless Tier.
```

File: tidb-cloud/serverless-tier-faqs.md (hunk 1 above)
Comment (SunRunAway) [reply tidb-cloud/serverless-tier-faqs.md:22]: 下面有link呀。

File: tidb-cloud/serverless-tier-faqs.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+- For each TiDB Cloud account, you can create one complimentary Serverless Tier cluster to use during the beta phase. You need to delete the existing Serverless Tier cluster then create a new one.
+- Each Serverless Tier cluster has the following limitations:
+    - The storage size is limited to 5 GiB (logical size) of OLTP storage and 5 GiB of OLAP storage.
+    - The compute resource is limited to 1 vCPU and 1 GiB RAM.
+- There are some features that Serverless Tier does not support or partially supports. See [Serverless Tier Limitations](/tidb-cloud/serverless-tier-limitations.md) for details.
+
+### Is Serverless Tier free during beta?
+
+Yes. Serverless Tier cluster is free to use during beta phase. When Serverless Tier goes GA, we will still offer a free starter plan and have a usage-based billing plan for additional resources and maintain higher performance.
```
[the remark this answers is not in the document]
Comment (SunRunAway) [reply tidb-cloud/serverless-tier-faqs.md:24]: 我又改了一下

File: tidb-cloud/serverless-tier-limitations.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
+---
+title: Serverless Tier Limitations
```
[the remark this answers is not in the document]
Comment (SunRunAway) [reply tidb-cloud/serverless-tier-limitations.md:2]: Done

File: tidb-cloud/serverless-tier-limitations.md (hunk 4 above)
Comment (SunRunAway) [reply tidb-cloud/serverless-tier-limitations.md:23]: I think a simple description is ok. Don't make people consider a lot here.

File: tidb-cloud/serverless-tier-limitations.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+We are constantly filling in the feature gaps between Serverless Tier and Dedicated Tier. If you require these features or capabilities in the gap, use [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) or [contact us](https://www.pingcap.com/contact-us/?from=en) for a feature request.
+
+## General limitations
+
+- For each TiDB Cloud account, you can create one complimentary Serverless Tier cluster during the beta phase. To create a new Serverless Tier cluster, you need to delete the existing one first.
+- Each Serverless Tier cluster has the following limitations:
+    - The storage size is limited to 5 GiB (logical size) of OLTP storage and 5 GiB (logical size) of OLAP storage.
+    - The compute resource is limited to 1 vCPU and 1 GiB RAM.
+    - **Note**: In the coming months, we intend to offer a usage-based billing plan for additional resources and higher performance, while still keeping offering the free starter plan. After the coming releases, the limitation of the free serverless tier may be changed.
```
[the remark this answers is not in the document]
Comment (qiancai) [reply tidb-cloud/serverless-tier-limitations.md:20]: Do we need to use the Markdown Note format for this line?

[the remark this answers is not in the document]
Comment (qiancai) [reply tidb-cloud/serverless-tier-limitations.md:20]: For "may" or "might", see https://developers.google.cn/style/word-list#may.

File: tidb-cloud/serverless-tier-faqs.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+
+Serverless Tier is in beta while we continuously add new features and improve existing features before it becomes generally available. We do not provide SLA for beta products. Therefore, Serverless Tier should **NOT** be used in production currently.
+
+### What are the limitations of a Serverless Tier cluster when in free beta?
+
+- For each TiDB Cloud account, you can create one complimentary Serverless Tier cluster during the beta phase. To create a new Serverless Tier cluster, you need to delete the existing one first.
+- Each Serverless Tier cluster has the following limitations:
+    - The storage size is limited to 5 GiB (logical size) of OLTP storage and 5 GiB of OLAP storage.
+    - The compute resource is limited to 1 vCPU and 1 GiB RAM.
+    - **Note**: In the coming months, we intend to offer a usage-based billing plan for additional resources and higher performance, while still keeping offering the free starter plan. After the coming releases, the limitation of the free serverless tier may be changed.
```
[the remark this answers is not in the document]
Comment (SunRunAway) [reply tidb-cloud/serverless-tier-faqs.md:36]:
```suggestion
    - **Note**: In the coming months, we intend to offer a usage-based billing plan for additional resources and higher performance, while still keeping offering the free starter plan. In the coming releases, the limitations of the free serverless tier might be changed.
```

**Review by qiancai — APPROVED** [review]
LGTM

**Review by sunxiaoguang — APPROVED** [review]

**Comment by sunxiaoguang** [comment]
LGTM

File: tidb-cloud/serverless-tier-faqs.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+
+Serverless Tier is in beta while we continuously add new features and improve existing features before it becomes generally available. We do not provide SLA for beta products. Therefore, Serverless Tier should **NOT** be used in production currently.
+
+### What are the limitations of a Serverless Tier cluster in free beta?
+
+- For each TiDB Cloud account, you can create one complimentary Serverless Tier cluster during the beta phase. To create a new Serverless Tier cluster, you need to delete the existing one first.
+- Each Serverless Tier cluster has the following limitations:
+    - The storage size is limited to 5 GiB (logical size) of OLTP storage and 5 GiB of OLAP storage.
+    - The compute resource is limited to 1 vCPU and 1 GiB RAM.
+    - **Note**: In the coming months, we intend to offer a usage-based billing plan for additional resources and higher performance, while still keeping offering the free starter plan. In the coming releases, the limitations of the free serverless tier might be changed.
```
Comment (ran-huang) [inline tidb-cloud/serverless-tier-faqs.md:36]:
```suggestion
    - **Note**: In the coming months, we intend to offer a usage-based billing plan for additional resources and higher performance, while still keeping offering the free starter plan. In the coming releases, the limitations of the free Serverless Tier might be changed.
```

File: tidb-cloud/serverless-tier-faqs.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+## Security FAQs
+
+### Is my Serverless Tier shared or dedicated?
+
+The serverless technology is designed for multi-tenancy and the resources used by all clusters are shared. To get managed TiDB service with isolated infrastructure and resources, you can upgrade it to the [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier).
+
+### How does TiDB Serverless Tier ensure security?
+
+- Your connections are encrypted by Transport Layer Security (TLS). For more information about using TLS to connect to Serverless Tier, see [Secure Connections to Serverless Tier Clusters](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md).
+- All persisted data on Serverless Tier is encrypted-at-rest using the cloud provider's tool that your cluster is running in.
```
Comment (ran-huang) [inline tidb-cloud/serverless-tier-faqs.md:56]:
```suggestion
- All persisted data on Serverless Tier is encrypted-at-rest using the tool of the cloud provider that your cluster is running in.
```

File: tidb-cloud/serverless-tier-faqs.md [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+
+## Maintenance FAQ
+
+### Can I upgrade the version of TiDB that my cluster is running on?
+
+No, Serverless Tier clusters are upgraded automatically as we release new TiDB versions on TiDB Cloud. You can see what version of TiDB your cluster is running in the [TiDB Cloud Console](https://tidbcloud.com/console/clusters) or in the latest [release note](https://docs.pingcap.com/tidbcloud/release-notes). Alternatively, you can also connect to your cluster and use `SELECT version()` or `SELECT tidb_version()` to check the TiDB version.
```
Comment (ran-huang) [inline tidb-cloud/serverless-tier-faqs.md:62]:
```suggestion
No. Serverless Tier clusters are upgraded automatically as we roll out new TiDB versions on TiDB Cloud. You can see what version of TiDB your cluster is running in the [TiDB Cloud Console](https://tidbcloud.com/console/clusters) or in the latest [release note](https://docs.pingcap.com/tidbcloud/release-notes). Alternatively, you can also connect to your cluster and use `SELECT version()` or `SELECT tidb_version()` to check the TiDB version.
```

File: tidb-cloud/serverless-tier-limitations.md [hunk 14]
```diff
@@ -0,0 +1,38 @@
...
+
+We are constantly filling in the feature gaps between Serverless Tier and Dedicated Tier. If you require these features or capabilities in the gap, use [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) or [contact us](https://www.pingcap.com/contact-us/?from=en) for a feature request.
+
+## General limitations
+
+- For each TiDB Cloud account, you can create one complimentary Serverless Tier cluster during the beta phase. To create a new Serverless Tier cluster, you need to delete the existing one first.
+- Each Serverless Tier cluster has the following limitations:
+    - The storage size is limited to 5 GiB (logical size) of OLTP storage and 5 GiB (logical size) of OLAP storage.
+    - The compute resource is limited to 1 vCPU and 1 GiB RAM.
+    - **Note**: In the coming months, we intend to offer a usage-based billing plan for additional resources and higher performance, while still keeping offering the free starter plan. In the coming releases, the limitations of the free serverless tier might be changed.
```
Comment (ran-huang) [inline tidb-cloud/serverless-tier-limitations.md:20]:
```suggestion
    - **Note**: In the coming months, we intend to offer a usage-based billing plan for additional resources and higher performance, while still keeping offering the free starter plan. In the coming releases, the limitations of the free Serverless Tier might be changed.
```

File: tidb-cloud/serverless-tier-limitations.md [hunk 15]
```diff
@@ -0,0 +1,38 @@
...
+- No "IP Access List" support.
+
+## Backup and Restore
+
+- [Backup and Restore](/tidb-cloud/backup-and-restore.md) are not supported for Serverless Tier currently.
+
+## Monitoring
+
+- [Built-in Monitoring](/tidb-cloud/built-in-monitoring.md) is currently not available for Serverless Tier.
+- [Third-party Monitoring integrations](/tidb-cloud/third-party-monitoring-integrations.md) is currently not available for Serverless Tier.
```
Comment (ran-huang) [inline tidb-cloud/serverless-tier-limitations.md:34]:
```suggestion
- [Third-party Monitoring integrations](/tidb-cloud/third-party-monitoring-integrations.md) are currently not available for Serverless Tier.
```

File: tidb-cloud/serverless-tier-limitations.md [hunk 16]
```diff
@@ -0,0 +1,38 @@
...
+
+## Others
+
+- [Open API](/tidb-cloud/api-overview.md) and [Terraform Integration](/tidb-cloud/terraform-tidbcloud-provider-overview.md) are currently not available for Serverless Tier.
```
Comment (ran-huang) [inline tidb-cloud/serverless-tier-limitations.md:38]:
```suggestion
- [TiDB Cloud API](/tidb-cloud/api-overview.md) and [Terraform Integration](/tidb-cloud/terraform-tidbcloud-provider-overview.md) are currently not available for Serverless Tier.
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

**Pull request merged**